base/array.jl

```diff
@@ -320,6 +320,8 @@ oneunit(x::AbstractMatrix{T}) where {T} = _one(oneunit(T), x)
 
 convert(::Type{Vector}, x::AbstractVector{T}) where {T} = convert(Vector{T}, x)
 convert(::Type{Matrix}, x::AbstractMatrix{T}) where {T} = convert(Matrix{T}, x)
+convert(::Type{Matrix}, x::Vector{T}) where {T} = Matrix(reshape(x, :, 1))
+convert(::Type{Matrix}, x::RowVector{T,S}) where {T,S} = Matrix(reshape(x, 1, :))
```

Comment:
@yuyichao I'm trying to figure out what you mean by the vector is not resizeable after a `reshape()`:

```
julia> x = Vector(1:4)
4-element Array{Int64,1}:
 1
 2
 3
 4

julia> y = reshape(x, 2, 2)
2×2 Array{Int64,2}:
 1  3
 2  4

julia> z = reshape(x, 1, 4)
1×4 Array{Int64,2}:
 1  2  3  4

julia> w = reshape(y, 1, 4)
1×4 Array{Int64,2}:
 1  2  3  4
```

Everything seems to "just work".  What makes something not resizeable?